.stylelintrc

```diff
@@ -0,0 +1,147 @@
+{
+  "rules": {
+    "at-rule-empty-line-before": [ "always", {
+      except: [
+        "blockless-after-same-name-blockless",
+        "first-nested",
```

Comment:
Doesn't look like theres a rule for that. Not a lot of cases where this happens, but preference would be to go first.